README.md

```diff
@@ -146,8 +146,9 @@ Verifying:
 <!-- @begin-sigstore-verify-help@ -->
 ```
 usage: sigstore verify [-h] [--certificate FILE] [--signature FILE]
-                       [--cert-email EMAIL] [--cert-oidc-issuer URL]
-                       [--staging] [--rekor-url URL]
+                       [--bundle FILE] [--cert-email EMAIL]
```

Comment:
nit: We probably should name this something else like `rekor-bundle` to avoid confusion when the "sigstore bundle" spec is completed.